proto/toit/model/data.proto

```diff
@@ -75,6 +76,9 @@ message LogData {
     STACK_TRACE = 4;
     BOOT = 5;
     SHUTDOWN = 6;
+    CONNECTED = 7;
+    DISCONNECTED = 8;
```

Comment:
CONNECT, CONNECT_FAIL, DISCONNECT maybe to be "consistent" with BOOT (not BOOTED) and SHUTDOWN.